metals/src/main/scala/scala/meta/internal/metals/ForwardingMetalsBuildClient.scala

```diff
@@ -52,6 +52,14 @@ final class ForwardingMetalsBuildClient(
   val updatedTreeViews: ju.Set[BuildTargetIdentifier] =
     ConcurrentHashSet.empty[BuildTargetIdentifier]
 
+  def buildHasErrors(buildTargetId: BuildTargetIdentifier): Boolean = {
+    buildTargets
+      .buildTargetTransitiveDependencies(buildTargetId)
+      .exists(id => hasReportedError.contains(id))
```

Comment:
if you don't find it less readable you can do `exists(hasReportedError.contains)`

Reply:
It's a java set, so I can't do it the other way.

Reply:
changed to `exists(hasReportedError.contains(_))`